### PR TITLE
[SETUPAPI] Improve SetupRemoveFromSourceListA/W exports CORE-16968

### PR DESCRIPTION
**Pull request by oleg-dubinskiy**
## Purpose

Improve exports for SetupRemoveFromSourceListA and SetupRemoveFromSourceListW functions in setupapi.dll by replacing `stub` with `stdcall -stub`. Also add functions arguments according to MSDN: https://docs.microsoft.com/en-us/windows/win32/api/setupapi/nf-setupapi-setupremovefromsourcelista, https://docs.microsoft.com/en-us/windows/win32/api/setupapi/nf-setupapi-setupremovefromsourcelistw, they are required by `stdcall -stub`.
It fixes EXCEPTION_WINE_STUB during installation the manufacturer's supplied software of USB wireless dongle D-LINK DWA-123, so now it does no longer fail (as confirmed by reporter).

JIRA issue: [CORE-16968](https://jira.reactos.org/browse/CORE-16968)

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
Isn't it something to be reported to WINE too ?
Refer to : https://github.com/wine-mirror/wine/blob/master/dlls/setupapi/setupapi.spec

**Comment by oleg-dubinskiy** [comment]
> Isn't it something to be reported to WINE too ?

Yes, but since our setupapi is forked from Wine, not winesynced, I think it is not urgent right now.

File: dll/win32/setupapi/setupapi.spec [hunk 1]
```diff
@@ -502,8 +502,8 @@
 @ stub SetupRemoveFileLogEntryW
 @ stub SetupRemoveFromDiskSpaceListA
 @ stub SetupRemoveFromDiskSpaceListW
-@ stub SetupRemoveFromSourceListA
-@ stub SetupRemoveFromSourceListW
+@ stdcall -stub SetupRemoveFromSourceListA(long ptr)
+@ stdcall -stub SetupRemoveFromSourceListW(long ptr)
```
Comment (SergeGautherie) [inline dll/win32/setupapi/setupapi.spec:506]:
```suggestion
@ stdcall -stub SetupRemoveFromSourceListA(long str)
@ stdcall -stub SetupRemoveFromSourceListW(long wstr)
```

Comment (oleg-dubinskiy) [reply dll/win32/setupapi/setupapi.spec]: Committed, thanks.

**Pull request merged**